test/configCases/plugins/banner-plugin/index.js

```diff
@@ -0,0 +1,14 @@
+it("should contain banner in bundle0 chunk", function() {
+	var fs = require("fs");
+	var source = fs.readFileSync(__filename, "utf-8");
+	source.should.containEql("A test value");
```

Comment:
this is weak test case because it may match against the line itself: `source.should.containEql("A test value");`

To amend, consider using regex: https://github.com/Dashed/webpack/blob/4024689e6345a54a0c243519d8057627da784266/test/configCases/plugins/uglifyjs-plugin/index.js#L4-L6
 consider using